Add widget to show repos missing .about.yml

monfresh opened the pull request:
To track adoption of the `.about.yml` standard, we thought it would be helpful to add a widget that shows which repos still have not added it.

In order to make it easy to do this with the inherited code that we forked, I made the `ORGS` env var singular, since we're only using this project for one GitHub organization.

I also moved the GitHub stats widgets to their own page at /github_stats to reduce clutter on the /default endpoint.

Closes #15.

monfresh committed:
Add widget to show repos missing .about.yml

To track adoption of the `.about.yml` standard, we thought it would be helpful to add a widget that shows which repos still have not added it.

In order to make it easy to do this with the inherited code that we forked, I made the `ORGS` env var singular, since we're only using this project for one GitHub organization.

I also moved the GitHub stats widgets to their own page at /github_stats to reduce clutter on the /default endpoint.

Closes #15.

mbland commented on dashboards/default.erb:
```diff
@@ -1,41 +1,19 @@
-<% content_for(:title) { "Github Open Source Dashboard" } %>
+<% content_for(:title) { "18F Projects Health Dashboard" } %>
```
Can we replace the hardcoded `18F` with `#{ENV['ORG']}`?

monfresh replied:
Yep! Good catch!

mbland replied:
Do you want me to point it out everywhere, or is this one catch enough?

monfresh replied:
I'll do a "find everywhere" for it.

mbland commented on spec/jobs/repos_without_about_yml_spec.rb:
```diff
@@ -0,0 +1,34 @@
+    job.trigger_block
+  end
+
+  it 'calls send_event with expected items' do
```
Wish I knew how to help you here... Any pointers to how to approach this? This framework is new to me.

monfresh replied:
I asked for help yesterday on the Dashing repo. Haven't heard back yet. Basically, I need to figure out what object is receiving `send_event`. I thought it would be the Sinatra Application, but `expect(Sinatra::Application).to receive(:send_event)` did not work. See https://github.com/Shopify/dashing/issues/615

monfresh committed:
Use ENV['ORG'] in HTML page templates

This allows other people who might fork or clone this project to have their organization name displayed in the page headers and titles.

monfresh commented:
@mbland I just pushed a new commit that fixes the org name in the HTML. Once this is merged, I'll submit a new PR for the optimizations that will allow us to run the app on Cloud Foundry.

mbland commented on spec/support/fake_github_api.rb:
```diff
@@ -0,0 +1,23 @@
+  def json_response(response_code, file_name)
+    content_type :json
+    status response_code
+    File.open(File.dirname(__FILE__) + '/fixtures/' + file_name, 'rb').read
```
I think you can do just `File.read` instead of `File.open().read`. It seems `File.read` supports the `'rb'` options as well.

mbland commented:
Made just one more minor suggestion, but LGTM. Feel free to merge with or without accepting the suggestion.

monfresh commented:
Yep, they both work, although I think with `File.read`, you have to use `mode: 'rb'` to specify the mode. I'll leave it as is for now. Thanks for the review!
